### PR TITLE
Add `ecdsa-rdfc-2019`, `eddsa-rdfc-2022` and `ecdsa-sd-2023` to supported cryptosuites.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Comment by JSAssassin** [comment]
@dlongley  The verification tests for  [VC signed with ecdsa-sd-2023](https://github.com/digitalbazaar/bedrock-vc-verifier/blob/f44ecb9129c1825dfd8d39741f608f7c14f1e1c9/test/mocha/mock-credentials.json#L265-L301) is currently failing, this is the error I am getting https://gist.github.com/JSAssassin/91efb24233792093b7a459893cac4c8f.  Could you please guide me on how I can generate a derived proof? Does `bedrock-vc-issuer` need to be updated to create vcs with derived proofs?

I also want to point out that the older credentials using `ecdsa-2019` and `eddsa-2022` with `data-integrity/v1` context are no longer passing verification tests. This is the error I am getting https://gist.github.com/JSAssassin/41a6a551af7a130c5492e8030a75a5ac. I had to update them to use `data-integrity/v2` context to make the tests pass, is that expected behavior?

**Comment by dlongley** [comment]
@JSAssassin, I'll respond out of band on some of the questions above for quicker discussion.

**Review by dlongley — APPROVED** [review]

**Comment by JSAssassin** [comment]
NOTE: All tests are passing locally.

File: test/package.json [hunk 1]
```diff
@@ -28,13 +28,11 @@
     "@bedrock/jsonld-document-loader": "^4.0.0",
     "@bedrock/kms": "^14.0.0",
     "@bedrock/kms-http": "^18.0.0",
-    "@bedrock/ledger-context": "^24.0.0",
     "@bedrock/meter": "^5.0.0",
     "@bedrock/meter-http": "^12.0.0",
     "@bedrock/meter-usage-reporter": "^9.0.0",
     "@bedrock/mongodb": "^10.0.0",
     "@bedrock/multikey-context": "^2.0.0",
-    "@bedrock/oauth2-verifier": "^2.0.0",
```
Comment (dlongley) [inline test/package.json:37]: This is a required peer dependency of `@bedrock/service-core` so it should not be removed.

File: test/package.json [hunk 2]
```diff
@@ -28,13 +28,11 @@
     "@bedrock/jsonld-document-loader": "^4.0.0",
     "@bedrock/kms": "^14.0.0",
     "@bedrock/kms-http": "^18.0.0",
-    "@bedrock/ledger-context": "^24.0.0",
```
Comment (dlongley) [inline test/package.json:31]: This is a required peer dependency of `@bedrock/veres-one-context` so it should not be removed.

**Review by dlongley — APPROVED** [review]
Approving -- but some peer deps that were removed as unnecessary need to be added back. I'm not sure what tool / command is being used to determine whether those are needed or not -- but it's probably failing us in some way. Those packages are required peer dependencies.

File: test/package.json (hunk 1 above)
Comment (JSAssassin) [reply test/package.json:37]: I've reverted back the test deps removal here [fd3bbbe](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/55/commits/fd3bbbe89cb8374622ab366d4c0b70df22620185). Not sure why but when I used `npx depcheck` it flagged `@bedrock/service-core` and `@bedrock/ledger-context` as unused packages and removal of those packages didn't cause any issues during installation or when running test.

**Pull request merged**